CMakeLists.txt

```diff
@@ -26,14 +26,14 @@ if(NOT DEFINED AER_THRUST_BACKEND AND DEFINED ENV{AER_THRUST_BACKEND})
 endif()
 
 if(AER_THRUST_BACKEND STREQUAL "CUDA")
-  include(nvcc_add_compiler_options)
-  set(CUDA_FOUND TRUE)
-  #include(FindCUDA) # for cuda_select_nvcc_arch_flags, CUDA_FOUND
-  include(FindCUDA/select_compute_arch)
-  enable_language(CUDA)
+	include(nvcc_add_compiler_options)
+	set(CUDA_FOUND TRUE)
+	#include(FindCUDA) # for cuda_select_nvcc_arch_flags, CUDA_FOUND
+	include(FindCUDA/select_compute_arch)
+	enable_language(CUDA)
 else()
-  # idiosyncrasy of CMake that it still creates a reference to this
-  set(CMAKE_CUDA_COMPILE_WHOLE_COMPILATION "")
+	# idiosyncrasy of CMake that it still creates a reference to this
+	set(CMAKE_CUDA_COMPILE_WHOLE_COMPILATION "")
 endif()
```

Comment:
Here and most of the changes below seem like unrelated whitespace changes. It'd probably be better to remove this from this already large PR to make it easier for people to bisect and review